### ISSUE TITLE
how to set the `--wallet` command line param in the docker image

### ISSUE THREAD

**Issue by djpnewton** [ElementsProject/lightning#5272]
I would like to use the new `--wallet` param to make backups.

I have updated to the new v0.11 docker image (elementsproject/lightningd:v0.11.0.1) but I havent figured out how I can set the --wallet parameter?

**Comment by vincenzopalazzo** [comment]
I agree! in this case, we should have some smart bash script that recognizes if we are running a lightning-cli command or lightning, something that I have done in https://github.com/clightning4j/clightning4j-node/blob/main/v0.11.0/entrypoint.sh but not in bash

**Comment by cdecker** [comment]
Notice that the `$@` should forward any argument to the entrypoint script to the `lightningd` process, so ideally you'd only need `docker run lightningd --wallet=... --otheroptions`, no need for wrappers and similar logic.

Now, I'm not sure that this works given the quotation marks around the `$@`, and we should remove them if it is interfering, but the idea pretty much is that you can call the docket image just as if it were a normal binary

**Comment by sangbida** [comment]
This issue is being closed as part of issue cleanup.

**Reason**: No recent activity (last updated 1309 da

If you believe this issue should remain open, please comment and we can reopen it.
